cmd/manager/main.go

```diff
@@ -4,31 +4,44 @@ import (
 	"flag"
 	"runtime"
 
-	"github.com/golang/glog"
 	"github.com/openshift/cluster-autoscaler-operator/pkg/operator"
 	"github.com/openshift/cluster-autoscaler-operator/pkg/version"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/klog"
 )
 
 func printVersion() {
-	glog.Infof("Go Version: %s", runtime.Version())
-	glog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
-	glog.Infof("Version: %s", version.String)
+	klog.Infof("Go Version: %s", runtime.Version())
+	klog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	klog.Infof("Version: %s", version.String)
 }
 
 func main() {
+	flag.Set("alsologtostderr", "true")
 	flag.Parse()
+
+	klogFlags := flag.NewFlagSet("klog", flag.ExitOnError)
+	klog.InitFlags(klogFlags)
+
+	// Sync the glog and klog flags.
+	flag.CommandLine.VisitAll(func(f1 *flag.Flag) {
+		f2 := klogFlags.Lookup(f1.Name)
+		if f2 != nil {
+			value := f1.Value.String()
+			f2.Value.Set(value)
+		}
+	})
+
```

Comment:
Should be safe to just use:
```go
flag.Set("logtostderr", "true")
klog.InitFlags(nil)
flag.Parse()
```

Reply:
This is from the example in the klog repo and how things like the cluster-api tools initialize things.

Reply:
No objection here, just noticed we no longer have to use that hack of iterating through the flags. So, it's worth simplifying it and testing if all the flags are still available and properly propagated to vendored glog.